sqlmesh/core/macros.py

```diff
@@ -872,7 +872,7 @@ def star(
         for excluded in exclude.expressions or except_.expressions
     }
     quoted = quote_identifiers.this
-    table_identifier = alias.name or relation.name
+    table_identifier = normalize_identifiers(alias.name or relation.name, dialect=evaluator.dialect).name
```

Comment:
Actually, I thought about this again and we do need to refactor this a bit. If a user passes a quoted name as an arg, you'll need to preserve them in the objects passed to the `normalize_identifiers` call, otherwise you risk normalizing a case-sensitive identifier.

I think this is what we should do instead:

```python
    table_identifier = normalize_identifiers(
        alias if alias.name else relation, dialect=evaluator.dialect
    ).name
```

Reply:
Made the suggested change and added a couple more test cases to cover those scenarios. Seems to be working!

I put this in draft because I was thinking that might avoid the CI checks for now. Was having a few hurdles getting the `make style` to run. But it didn't.

If it doesn't pass CI this time, I will get that fixed later today. Appreciate the guidance!